akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ReplicatedEventSourcingSpec.scala

```diff
@@ -560,9 +560,7 @@ class ReplicatedEventSourcingSpec
         probe.expectMessageType[State].all.toSet shouldEqual Set("from r1", "from r2")
       }
       val intercepted = interceptProbe.receiveMessages(2)
-      intercepted.toSet shouldEqual Set(
-        Intercepted(ReplicaId("R1"), 2L, "from r1"),
-        Intercepted(ReplicaId("R2"), 2L, "from r2"))
```

Comment:
is it the seq nbrs that can end up different?

Reply:
Yeap